README.md

```diff
@@ -53,6 +53,7 @@ Below is the list of variables you can redefine in your playbook to customize st
 | **st2web**
 | `st2web_ssl_certificate`     | `null` | String with custom SSL certificate (`.crt`). If not provided, self-signed certificate will be generated.
 | `st2web_ssl_certificate_key` | `null` | String with custom SSL certificate secret key (`.key`). If not provided, self-signed certificate will be generated.
+| `st2web_nginx_conf`     | `/usr/share/doc/st2/conf/nginx/st2.conf` | String with path to nginx conf file.
```

Comment:
I'm suspecting that exposing `st2.conf` nginx config this way will lead to confusion or wrong/inappropriate use.

Also taking the var description `String with path to nginx conf file.`, as as user I'd also wonder why this setting is set to `/usr/share/doc/..` and why it's path to nginx config file or how nginx will read config from that dir (from the var description).

---

In a normal situation I'd say transferring custom st2.conf nginx config to some specific place like `/tmp/st2.conf` and overriding this setting in `ansible-st2` looks a bit strange as you'll need another Ansible task to upload that file to VM prior to running ansible-st2.

Instead of conf file path on a VM, how about exposing this var as nginx raw config/text?
This would be somewhat similar to `st2web_ssl_certificate_key` and `st2web_ssl_certificate` approach.

Reply:
@arma So are you saying, 

1. leave it the way it was if null, and write it if not null?
Or 
2. Ship the standard file in ansible, and if that var is set, then copy the contents?

Reply:
I found an approach.